inspirehep/base/templates/format/record/Inspire_Default_HTML_brief.tpl

```diff
@@ -110,8 +125,8 @@
               {% if record.get('earliest_date') %}
                 <i class="glyphicon glyphicon-calendar"></i> {{ record.get('earliest_date').split('-')[0] }}<br/>
               {% endif %}
-              {% if  record.get('citation_count') > 0  %}
-                <i class="fa fa-quote-left"></i><span><a href="/search?p=refersto:{{ record.get('control_number') }}"> Cited {{ record.get('citation_count') }} times</a></span><br/>
+              {% if record.get('citation_count') > 0  %}
+                <i class="fa fa-quote-left"></i><span><a href="/record/{{ record.get('control_number') }}#citations"  target="_blank"> Cited {{ record.get('citation_count') }} times</a></span><br/>
```

Comment:
Doing a search is the intended behaviour here. We will keep the functionality as is now - Check clicking on a Citation count here https://inspirehep.net/search?p=author%3Arandall+|+author%3Asundrum+cited%3A300-%3E1350&rm=citation